sanguk/app.js

```diff
@@ -32,6 +32,22 @@ app.get("/ping", (req,res)=>{
     res.json({message : "pong"});
 })
 
+app.post("/users", async (req,res,next)=>{
```

Comment:
```suggestion
app.post("/users/signup", async (req,res,next)=>{
```
주소를 좀 더 정확하게 적어주시면 좋을 것 같아요~!

Reply:
아 저거는 생각을 미처 못 한거 같습니다. 다음에 과제 할때 한번 생각하고 업데이트 하도록 하겠습니다.^^